builtin/k8s/platform.go

```diff
@@ -760,6 +771,28 @@ func (p *Platform) resourceDeploymentCreate(
 		return false, nil
 	})
 	if err != nil {
+		step.Update("Error detected waiting for Deployment to start.")
+		step.Abort()
+
+		ui.Output("The following is events for pods observed while attempting to start the Deployment", terminal.WithWarningStyle())
+
+		for uid, name := range podsSeen {
+			sel := ev.GetFieldSelector(nil, nil, nil, (*string)(&uid))
+
+			events, err := ev.List(ctx, metav1.ListOptions{
+				FieldSelector: sel.String(),
+			})
+			if err == nil {
+				ui.Output("Events for %s", name, terminal.WithHeaderStyle())
+				for _, ev := range events.Items {
+					if ev.Type == "Normal" {
+						continue
+					}
+					ui.Output("  %s: %s (%s)", ev.Type, ev.Message, ev.Reason)
```

Comment:
I also noticed this change in your demo, excited for this! 😄

Reply:
![bitmoji](https://sdk.bitmoji.com/render/panel/d080f5b6-bdfd-4482-bf01-52b398c8e790-8cb64c44-6a64-4f62-9642-460dd37039d6-v1.png?transparent=1&palette=1&width=246)